osu.Game/Audio/Effects/Filter.cs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using ManagedBass.Fx;
+using osu.Framework.Audio.Mixing;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+
+namespace osu.Game.Audio.Effects
+{
+    public class Filter : Component
+    {
+        public BQFType FilterType = BQFType.LowPass;
+        public float SweepCutoffStart = 2000;
+        public float SweepCutoffEnd = 150;
+        public float SweepDuration = 100;
+        public Easing SweepEasing = Easing.None;
```

Comment:
These shouldn't really be here. Instead a consume should just be able to call `filter.CutoffTo(newValue, time, Easing)` as per any other transform.